### PR TITLE
new kottans

### PR DESCRIPTION
**Pull request by BorysYurii**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,69 +1,23 @@
...
+  "name": "bitbybit",
+  "author": "bitbybit",
   "version": "1.0.0",
-  "scripts": {
-    "ghpages": "git subtree push --prefix build origin gh-pages",
-    "build": "gulp build --production",
-    "start": "gulp",
-    "lint": "eslint --cache --format=pretty \"**/*.js\"",
-    "lint-fix": "npm run lint -- --fix",
-    "prettier:fix": "prettier --ignore-path .gitignore \"**/*.+(js|json|scss)\" --write"
```
Comment (zonzujiro) [inline package.json:10]: Pls, do not remove these scripts. We need it :)

File: package.json [hunk 2]
```diff
@@ -1,69 +1,23 @@
 {
-  "name": "kottans_website",
+  "name": "bitbybit",
```
Comment (zonzujiro) [inline package.json:2]: `kottans_website` pls :)

File: assets/js/script.js [hunk 3]
```diff
@@ -1,125 +1,239 @@
...
-  mobileMenu.addEventListener('click', () => {
-    document.body.classList.toggle('active-menu');
-  })
+          let elIndex = thisEl.index + 1;
 
-  document.addEventListener('click', ev => {
-    const container = document.querySelector('.mob-menu, .mobile_menu_js');
+          document.querySelector('.comment_text_js.active').classList.remove('active');
+          document.querySelector('.comment_text_js:nth-child('+elIndex+'n)').classList.add('active');
+          document.querySelector('.comment_text_js:nth-child('+elIndex+'n)').style.display = "block"; // no animation
```
Comment (zonzujiro) [inline assets/js/script.js:123]: You can use ES6 syntax instead:

```suggestion
          document.querySelector('.comment_text_js.active').classList.remove('active');
          document.querySelector(`.comment_text_js:nth-child('${elIndex}'n)`).classList.add('active');
          document.querySelector(`.comment_text_js:nth-child('${elIndex}'n)`).style.display = "block"; // no animation
```

File: assets/js/script.js [hunk 4]
```diff
@@ -1,125 +1,239 @@
...
 
-    if (container !== ev.target && !container.contains(ev.target)) {
-      document.body.classList.remove('active-menu')
+          let img = 'url(' + thisEl.dataset.fullImg + ')';
```
Comment (zonzujiro) [inline assets/js/script.js:125]:
```suggestion
          let img = `url('${thisEl.dataset.fullImg}')`;
```

File: assets/js/script.js [hunk 5]
```diff
@@ -1,125 +1,239 @@
...
+          var activeEl = document.querySelector(".comment_js.active");
+          activeEl.classList.remove('active');
+
+          thisEl.classList.add('active');
+
+          document.querySelector('.comment_text_js.active').style.display = "none"; // no animation
+
+          for(i=0; i < comments.length; i++) {
+            comments[i].index = i;
+          }
```
Comment (zonzujiro) [inline assets/js/script.js:117]:
```suggestion
          comments.forEach((comment, i) => {
            comment.index = i;
          })
```

File: assets/js/script.js [hunk 6]
```diff
@@ -1,125 +1,239 @@
...
+
+    initMentors: () => {
+      let mentorTab = document.querySelectorAll('.tabs_js .mentors-block__tab');
+
+      mentorTab.forEach((item, i) => {
+        item.addEventListener('click', function(){
+          document.querySelector('.mentors-block__tab.active').classList.remove('active');
+          item.classList.add('active');
+          document.querySelector('.mentors-block__slider.active').classList.remove('active');
+          document.querySelector('.mentors-block__slider:nth-of-type(' + (i + 1) + ')').classList.add('active');
```
Comment (zonzujiro) [inline assets/js/script.js:144]: ES6 also

File: assets/js/script.js [hunk 7]
```diff
@@ -1,125 +1,239 @@
...
+        }
+      ]
+      });
+    },
+
+    initMentorsSlider: () => {
+      let mentorsSlider = document.querySelectorAll('.mentors_slider_js');
+
+      mentorsSlider.forEach((slider, i) => {
+        slider.insertAdjacentHTML('afterend', '<div class="dots_block_'+(i+1)+'"></div>');
```
Comment (zonzujiro) [inline assets/js/script.js:183]: ES6 strings

File: assets/js/script.js [hunk 8]
```diff
@@ -156,14 +270,13 @@ initMenu();
         var animateBlock = $('.ab');
 
         function animate() {
-          var winScroll = $(window).scrollTop();
           var animOffset = $(window).height() / 1.3;
 
           animateBlock.each(function () {
```
Comment (zonzujiro) [inline assets/js/script.js:275]: You can replace such loops with something like this:

```javascript
const animateBlocks = document.querySelectorAll('.ab');

animateBlocks.forEach(block => { ... })
```

**Comment by OleksiyRudenko** [comment]
Thank you for the site refactoring. Please make sure the intra-page anchors remain consistent (so that any published links remain actual).
I mean the following from the current edition:
image

File: html-dev/home/3-our-courses.html [hunk 9]
```diff
@@ -0,0 +1,723 @@
...
+          <div class="course__line">
+            <div class="buttons-block">
+              <a href="https://github.com/kottans/frontend" target="_blank" class="btn-default btn-default--big">
+                <span class="btn-default__inner">
+                  check details
+                </span>
+              </a>
+              <a href="https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw" target="_blank" class="btn-link icon-tg">
+                <div class="btn-link__inner">
+                  Questionariuam
```
Comment (AMashoshyna) [inline html-dev/home/3-our-courses.html:307]:
```suggestion
                  Questionaruam
```

File: html-dev/home/3-our-courses.html [hunk 10]
```diff
@@ -0,0 +1,723 @@
...
+          <div class="course__line">
+            <div class="buttons-block">
+              <a href="https://github.com/kottans/backend" target="_blank" class="btn-default btn-default--big">
+                <span class="btn-default__inner">
+                  check details
+                </span>
+              </a>
+              <a href="https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw" target="_blank" class="btn-link icon-tg">
+                <div class="btn-link__inner">
+                  Questionariuam
```
Comment (AMashoshyna) [inline html-dev/home/3-our-courses.html:647]:
```suggestion
                  Questionarium
```

**Pull request merged**